src/operator/windowTime.ts

```diff
@@ -147,52 +163,35 @@ class WindowTimeSubscriber<T> extends Subscriber<T> {
     return window;
   }
 
-  closeWindow(window: Subject<T>) {
+  closeWindow(window: Subject<T>): void {
     window.complete();
     const windows = this.windows;
     windows.splice(windows.indexOf(window), 1);
   }
 }
 
-interface TimeSpanOnlyState<T> {
-  window: Subject<any>;
-  windowTimeSpan: number;
-  subscriber: WindowTimeSubscriber<T>;
-}
-
-function dispatchWindowTimeSpanOnly<T>(this: Action<TimeSpanOnlyState<T>>, state: TimeSpanOnlyState<T>) {
+function dispatchWindowTimeSpanOnly<T>(this: Action<TimeSpanOnlyState<T>>, state: TimeSpanOnlyState<T>): void {
   const { subscriber, windowTimeSpan, window } = state;
   if (window) {
-    window.complete();
+    subscriber.closeWindow(window);
```

Comment:
yes, correct. previously timspan'ed window closed, but not teared down as same as default window behavior. Now both uses same teardown path. Sorry for creating commit include mixed changes, should I split up and update PR?